File: app/src/main/kotlin/com/jsj/kotlindemo/java/MainActivityJava.java
package com.jsj.kotlindemo.java;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

import com.jsj.kotlindemo.test.Rectangle;
import com.jsj.kotlindemo.test.StringFunctions;
import com.jsj.kotlindemo.viewpager.DetailsActivityKt;


/**
 * @author jsj
 * @since 17/8/10
 */

public class MainActivityJava extends AppCompatActivity{



    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Rectangle rectangle = new Rectangle(3,4);
        rectangle.isSquare();
        StringFunctions.joinToString();
        DetailsActivityKt.lastChar("kotlin");
        DetailsActivityKt.getLastChar("Kotlin");
        DetailsActivityKt.setLastChar(new StringBuilder(),'H');
    }



}

File: app/src/main/kotlin/com/jsj/kotlindemo/java/MainActivityJava.kt
package com.jsj.kotlindemo.java

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.TextView
import com.jsj.kotlindemo.R

/**
 * @author jsj
 * *
 * @since 17/5/20
 */

class MainActivityJava : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val textView = findViewById(R.id.tv_name) as TextView
        textView.text = "你好"

        val array = arrayOf(1, 2, 3)
    }

    /**
     * 设置年龄
     * @return
     */
    fun setAge(age: Int): Int {

        return age
    }
}
